app/src/main/java/net/squanchy/favorites/view/FavoritesItem.kt

```diff
@@ -11,7 +13,8 @@ sealed class FavoritesItem {
 
     data class Header(val date: LocalDate) : FavoritesItem() {
 
-        override val id = date.toDateTimeAtCurrentTime().millis // We're relatively sure this won't clash with event IDs
+        override val id = date.atTime(LocalTime.now())
+            .atZone(ZoneId.systemDefault()).toInstant().toEpochMilli() // We're relatively sure this won't clash with event IDs
```

Comment:
What about breaking it down? It is easy to miss what's going on in the chain otherwise

```suggestion
        // We're relatively sure this won't clash with event IDs
        override val id = date.atTime(LocalTime.now())
                    .atZone(ZoneId.systemDefault())
                    .toInstant()
                    .toEpochMilli()
```